Makefile

```diff
@@ -39,6 +39,24 @@ start: ## Start the development environment (use Docker)
 stop: ## Stop FreshRSS container if any
 	docker stop freshrss-dev
 
+###########
+## Tests ##
+###########
+.PHONY: install-test
+install-test: ## Install required packages for test purposes
+	mkdir -p bin/
+	wget -O bin/phpunit https://phar.phpunit.de/phpunit-7.phar
```

Comment:
Good point, this way my minor objection doesn't really apply.

Reply:
In the mean time, it's not very secure if we don't verify the binary, I'm a bit annoyed by this solution.

Reply:
Just stick an md5/sha1 on it?

Reply:
If we ship a hash, then it must be for a precise release

Reply:
Naturally, but that's just https://phar.phpunit.de/phpunit-7.5.9.phar with SHA256 `5404288061420c3921e53dd3a756bf044be546c825c5e3556dea4c51aa330f69`. See https://phar.phpunit.de/ for the full list.

Reply:
Done, good idea :+1:

The file is removed if the checksum fails. I also added a bit of documentation.